e2e/scenario_test.go

```diff
@@ -953,7 +953,6 @@ func runScenarioUbuntu2204GPU(t *testing.T, vmSize string) {
 				ValidateNvidiaModProbeInstalled(ctx, s)
 				ValidateKubeletHasNotStopped(ctx, s)
 				ValidateServicesDoNotRestartKubelet(ctx, s)
-				ValidateEnableNvidiaResource(ctx, s)
```

Comment:
I added these validators a few days before. Apparently it's been a wrong decision.